javascripts/background_scripts/first_run.js

```diff
@@ -74,25 +83,23 @@ var firstRun = {
 
       // Dissable the posting button by default if the user already has
       // the extension installed.
-      if ( ls.getItem("posting_content_server_url") !== undefined ) {
+      if (ls.getItem("posting_content_server_url") !== undefined) {
         ls.setItem("Options:DissableButton", "true");
       }
 
-      ls.setItem("glyph_color", Math.floor(Math.random()*16777215).toString(16));
+      ls.setItem("glyph_color", Math.floor(Math.random() * 16777215).toString(16));
       var glyph_cells = ((Math.random() < 0.5) ? "false" : "true");
-      for(var i = 0; i < 14; i++) {
+      for (var i = 0; i < 14; i++) {
         glyph_cells += "," + ((Math.random() < 0.5) ? "false" : "true");
       }
       ls.setItem("glyph_cells", glyph_cells);
     }
 
-    var runningVersion = firstRun.getPrivlyVersion();
```

Comment:
Always set the version here and remove the check for the stored version in firstRun.firstRun. Much of this version comparison logic is not necessary if we use the Chrome API to open the firstRun page.